xml/art_sle_ha_install_quick.xml

```diff
@@ -897,6 +915,16 @@ info: Received command test from &node1; on disk <replaceable>SBD</replaceable><
        <keycap function="control"/><keycap>C</keycap>
       </keycombo>.
    </para>
+   <para>
+    The above procedure is just a simple test to check, if the cluster moves
```

Comment:
The above procedure is just a simple test to check, if the cluster moves the virtual IP address to the right node. -> Rephrase to: The above procedure is just a simple test to check if the cluster moves the virtual IP address to the other node. A realistic test involves specific use cases and scenarios. Before using the cluster in a production environment, test it thoroughly according to your use cases.